### PR TITLE
Document tests/integration/_common.sh.

### PR DESCRIPTION
**Pull request by jsha**
This makes it much easier to run certbot during development.

### CODE REVIEW

File: docs/contributing.rst [hunk 1]
```diff
@@ -39,17 +39,17 @@ Then in each shell where you're working on the client, do:
 .. code-block:: shell
 
    source ./venv/bin/activate
+   . tests/integration/_common.sh
```
Comment (zjs) [inline docs/contributing.rst]: Should this use `source` instead of `.` for consistency with the above line? (Or vice versa.)

Comment (jsha) [inline docs/contributing.rst]: Good point, fixed.

**Comment by SwartzCr** [comment]
so would you recommend that @jsha remove that section?

**Comment by jsha** [comment]
One option would be to add an `export SERVER=https://acme-staging.api.letsencrypt.org/directory` to that section.

FWIW, I actually think it's much nicer to develop against a local Boulder, but I do see how that's one more step and probably most contributors are not doing it.

**Comment by zjs** [comment]
Maybe it would be worth having a sub-section about developing against a local Boulder?

That would give an "easy mode" (the instructions as they exist today) and an "advanced mode" (local Boulder, certbot_test).

That would allow developers to opt-in to the complexity.

I might normally be wary of having multiple ways of doing things, but it seems like there's value in having a low-friction process for first-time/one-time contributors and value in making repeat contributors as efficient as possible.

**Comment by jsha** [comment]
> That would give an "easy mode" (the instructions as they exist today) and an "advanced mode" (local Boulder, certbot_test).

I *think* the instructions as they exist today are not "easy mode" because you have to figure out a big long set of flags to pass, right? Or do you just run as root?

My goal with this change is to promote certbot_test (or something similar) as an "easy mode." If that means making the default instructions use the staging server, I'm fine with that.

FWIW, there's already a bit of a section about running against Boulder: https://certbot.eff.org/docs/contributing.html#integration.

**Comment by zjs** [comment]
> My goal with this change is to promote certbot_test (or something similar) as an "easy mode." If that means making the default instructions use the staging server, I'm fine with that.

I see. I think that makes sense.

> FWIW, there's already a bit of a section about running against Boulder: https://certbot.eff.org/docs/contributing.html#integration.

I think this is a good start that could be expanded to cover running manual tests against a local bounder as well.

**Pull request merged**